services/Makefile

```diff
@@ -127,8 +127,14 @@ publish: container
 update-center.json:
 	wget -O update-center.json https://updates.jenkins.io/update-center.actual.json
 
+update-center-experimental.json:
+	wget -O update-center-experimental.json https://updates.jenkins.io/experimental/update-center.actual.json
+
 propose-updates: update-center.json
-	node ./prepare-essentials propose-updates
+	node ./prepare-essentials propose-updates --uc ./update-center.json
+
+propose-experimental-updates: update-center-experimental.json
```

Comment:
Just offering an alternate target. Really this stuff should of course be replaced with usage of the updater logic from `incrementals-tools` so that we get a single consistent way to update everything to whatever is in `master`.